_includes/overall-schedule.md

```diff
@@ -17,11 +17,39 @@ Organizers will inform participants of the week schedule by email.
 
 | Week | Call | Date | Topic | Agenda |
 |------|------|------|-------|--------|
-{%- for w in schedule.weeks %}
+ {%- for w in schedule.weeks %}
 {%- capture w-desc %}**Week {{ w[0] }}** (start. {{ w[1].start }}){% endcapture %}
 {%- for c in w[1].calls %}
 {%- capture date %}{% if c.type != "Mentor-Mentee" %}{{ c.date }} {% if c.time %}([{{ c.time }} European Time](https://arewemeetingyet.com/Berlin/{{ c.date | date: "%Y-%m-%d" }}/{{ c.time }}/{{ cohort }}%20{{ c.type }}%20Call%20(Week%20{{ w[0] }}))){% endif %}{% endif %}{% endcapture %}
 | {{ w-desc }} | [{{ c.type }}](/{{ cohort }}#{{ c.type | downcase | remove: "(" | remove: ")" | remove: "@" | remove: ":" | remove: "," | replace: " ", "-" | remove: "&"  }}-calls) | {{ date }} | [**{{ c.title }}**](/{{ cohort }}/schedule#week-{{ w[0] }}) | {% if c.agenda %}{{ c.agenda }}{% endif %} |
 {%- assign w-desc = "" %}
-{%- endfor %}
-{%- endfor %}
+{%- endfor %} 
+{%- endfor %} 
+
+<script type="application/ld+json" >
+{
+  "@context": "https://schema.org",
+  "@id": "{{ site.url }}/{{ cohort }}/schedule",
+  "@type": "Course",
+  "dct:conformsTo": "https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08",
+  "description": "{%- for w in schedule.weeks %} {%- capture w-desc %}**Week {{ w[0] }}** (start. {{ w[1].start }}){% endcapture %} {%- for c in w[1].calls %}{%- capture date %} {% if c.type !='Mentors-Mentee'%} {{ c.date }} {% if c.time %} ([{{ c.time }} European Time] (https://arewemeetingyet.com/Berlin/{{ c.date | date: '%Y-%m-%d' }}/{{ c.time }}/{{ cohort }}%20{{ c.type }}%20Call%20(Week%20{{ w[0] }}))) {% endif %}{% endif %}{% endcapture %} | {{ w-desc }} | [{{ c.type }}](/{{ cohort }}#{{ c.type | downcase | remove: '(' remove: | remove:')'| remove: '@' | remove: ':' | remove: ',' | replace: ' ', '-' | remove: '&'}}-calls) | {{ date }} | [**{{ c.title }}**] (/{{ cohort }}/schedule#week-{{ w[0] }}) | {% if c.agenda %}{{ c.agenda }}{% endif %} |{%- assign w-desc = '' %} {%- endfor %} {%- endfor %}",
```

Comment:
The description here should be about OLS and the cohort not the content of every week

Reply:
You need to move this `for` loop in the `hasCourseInstance` (line 37). Every iteration in the loop is a week and for every week there should be calls. So for each call (so 2 loops) you need create a `CourseInstance` dictionary in the `hasCourseInstance` with different components `startDate` and other

Reply:
If the `for loop` is moved to line 37, it throws an error of unparsable structured data. Unless the `for loop` is a value of a property. I think I'm missing something just don't know what..

Reply:
You need to break the content of the for loop there as it extracts different content, separated by  `|`